css/properties/transition-duration.json

```diff
@@ -0,0 +1,123 @@
+{
+  "css": {
+    "properties": {
+      "transition-duration": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/transition-duration",
+          "support": {
+            "webview_android": {
```

Comment:
CSS Transitions have been unprefix in Chrome 26 (prefixed version kept): see https://blog.chromium.org/2013/02/chrome-26-beta-template-element.html

(please add extra unprefixed entry with true)